Content.Shared/_RMC14/Xenonids/Egg/EggRetriever/XenoEggRetrieverComponent.cs

```diff
@@ -0,0 +1,23 @@
+using Robust.Shared.Prototypes;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Content.Shared._RMC14.Xenonids.Egg;
+
+/// <summary>
+/// Allows a xeno to stash eggs into an internal inventory and bring them back out
+/// </summary>
+[RegisterComponent]
+public sealed partial class XenoEggRetrieverComponent : Component
+{
+    public const string EggContainerId = "eggs";
+
+    [DataField]
+    public EntProtoId EggPrototype = "XenoEgg";
+
+    [DataField]
+    public int MaxEggs = 7;
```

Comment:
Very minor but max eggs is 8 [can check caste code here](https://github.com/cmss13-devs/cmss13/blob/d226d88e7ddb38e1ad2169852f0f99682b44ee5a/code/modules/mob/living/carbon/xenomorph/castes/Carrier.dm#L101)

Reply:
How out of date is this wiki? 
Thanks for telling me.